openshift/tests-extension/.openshift-tests-extension/openshift_payload_olmv1.json

```diff
@@ -9,7 +9,7 @@
       "isolation": {}
     },
     "source": "openshift:payload:olmv1",
-    "lifecycle": "informing",
+    "lifecycle": "blocking",
```

Comment:
Is it producing signal to Sippy? 
Could we move all tests that will become a blocking to be centralised?

In this case, can we move any QE shaped test to : https://github.com/openshift/operator-framework-operator-controller/blob/main/openshift/tests-extension/test/

Also, are those scenarios that are not covered yet? 
It would be great we ensure that we have not many tests covering the same. 

Otherwise, it might start be hard for we keep things maintained.